lib/checkPullRequestReview.js

```diff
@@ -127,9 +127,10 @@ const handleApprovalByAllReviewers = async (context, pullRequest) => {
     const commentBody =
       'Hi @' +
       projectOwner +
-      ', this PR is ready to be merged. Before you merge it, please ' +
-      'make sure that there are no pending comments that require ' +
-      "action from the author's end. Thanks!";
+      ', this PR is ready to be merged. Author of this PR ' +
+      'does not have permissions to merge this PR. Before you merge it, ' +
+      'please make sure that there are no pending comments that require ' +
+      'action from the author's end. Thanks!';
```

Comment:
@vojtechjelinek In this line, only double quote can be used as "author's" has single quote which will end the statement.